packages/office-ui-fabric-react/etc/office-ui-fabric-react.api.ts

```diff
@@ -11459,7 +11459,7 @@ class ScrollablePaneBase extends BaseComponent<IScrollablePaneProps, IScrollable
   // (undocumented)
   shouldComponentUpdate(nextProps: IScrollablePaneProps, nextState: IScrollablePaneState): boolean;
   // (undocumented)
-  sortSticky: (sticky: Sticky) => void;
+  sortSticky: (sticky: Sticky, reSort?: boolean | undefined) => void;

```

Comment:
maybe use "sortAgain" instead? it's kind of awkward to type something like reSort I think.